clientupdate/distsign/distsign.go

```diff
@@ -229,8 +229,6 @@ func (c *Client) Download(ctx context.Context, srcPath, dstPath string) error {
 	c.logf("Downloading %q", sigURL)
 	sig, err := fetch(sigURL, signatureSizeLimit)
 	if err != nil {
-		// Best-effort clean up of downloaded package.
-		os.Remove(dstPathUnverified)
```

Comment:
might need to update https://github.com/tailscale/tailscale/blob/87c5ad4c2c0e108e0c66cdb95f75ae5a5da72e31/clientupdate/clientupdate.go#L915 for people with frequent interrupted downloads and limited disk space

Reply:
Hm. Maybe only delete things with modtime over a day? Any thoughts?

Reply:
that, or glob for `*.unverified` in `distsign.Download` and delete everything with a prefix other than `dstPath`?